cmd/bedMerge/bedMerge.go

```diff
@@ -52,9 +52,9 @@ func bedMergeLowMem(infile string, outfile string, mergeAdjacent bool) {
 func bedMergeHighMem(infile string, outfile string, mergeAdjacent bool) {
 	var records []bed.Bed = bed.Read(infile)
 	var outList []bed.Bed
-	var currentMax bed.Bed = records[0]
 
 	bed.SortByCoord(records)
+	var currentMax bed.Bed = records[0]
```

Comment:
I approve this change but it seems like all the other changes are from the bedSubsetMatrix pull request. Should I not approve this (bedMergeBug) pull request now and wait until after bedSubsetMatrix has been merged?